### DISCUSSION TITLE
I changed the config, but I see no changes

### DISCUSSION THREAD

**Discussion by Airblader** [i3/i3#4506] [Q&A] [answered]
I have made some changes to my i3 config file. However, when I reload the configuration, nothing seems to happen.

**Comment by Airblader** [comment, marked as answer]
i3 looks for a config file in a few different places. Please make sure you're editing the correct file by looking at `i3 --more-version`. This will tell you the location of the config file currently being used:

```
$ i3 --more-version

…
Loaded i3 config: /home/me/.i3/config (…)
…
```

If this is correct, try completely logging out and back in as there are differences between `reload`, `restart` and a completely new session.

**Comment by EllaTheCat** [comment]
Regolith is quite a popular distribution that provides i3. Apropos of the i3 config file it does things slightly differently which has lead to confusion in the past over on reddit. 

 See https://regolith-linux.org/docs/interface/system/

 I don't use Regolith, this is just something I've learnt about. This is also a test posting following the PSA  over on reddit.
